test/rowSelection.test.tsx

```diff
@@ -10,7 +10,16 @@ interface Row {
   id: number;
 }
 
-const columns: readonly Column<Row>[] = [SelectColumn];
+const columns: readonly Column<Row>[] = [
+  SelectColumn,
+  {
+    key: 'name',
+    name: 'Name',
+    formatter() {
+      return null;
+    }
```

Comment:
The default formatter is good enough.
```suggestion
    name: 'Name'
```